requirements/requirements.txt

```diff
@@ -36,19 +36,19 @@ pycrypto==2.6.1
 py-KISSmetrics==1.0.1
 https://github.com/dimagi/files/raw/master/python-requirements/pygooglechart-0.3.0.tar.gz
 python-dateutil==1.5
-python-magic==0.4.10
+python-magic==0.4.13
```

Comment:
no tag exists for 0.4.10...

https://github.com/ahupp/python-magic/compare/0.4.6...0.4.13